module/surveys/helper.js

```diff
@@ -495,9 +495,9 @@ module.exports = class SurveysHelper {
         } else {
           let survey = {};
 
-          if (userOrganisations.createdFor) {
-            survey['createdFor'] = userOrganisations;
-          }
+          // if (userOrganisations.createdFor) {
```

Comment:
remove commented code @praveenKDass if it is not required for future reference

Reply:
It might be needed in future if we are adding org @borkarsaish65